From: Rong Tao <rongtao@cestc.cn>
Subject: tools/vm/page_owner: ignore page_owner_sort binary
Date: Tue, 8 Nov 2022 19:53:48 +0800

page_owner_sort was introduced since commit 48c96a368579 ("mm/page_owner:
keep track of page owners"), and we should ignore it.

Link: https://lkml.kernel.org/r/tencent_F6CAC0ABE16839E2B2419BD07316DA65BB06@qq.com
Signed-off-by: Rong Tao <rongtao@cestc.cn>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/vm/.gitignore |    1 +
 1 file changed, 1 insertion(+)

--- a/tools/vm/.gitignore~tools-vm-page_owner-ignore-page_owner_sort-binary
+++ a/tools/vm/.gitignore
@@ -1,3 +1,4 @@
 # SPDX-License-Identifier: GPL-2.0-only
 slabinfo
 page-types
+page_owner_sort
_
